### PR TITLE
Transpose matrix

### PR DESCRIPTION
**Pull request by HBaena**
## Description

[FEATURE] Transpose matrix 

## What does your PR belong to?
- [ ] Website
- [ x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. 
- [ x] I have checked that the changes are working properly
- [ x] I have checked that there isn't any PR doing the same
- [ x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/transpose.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Make a transpose matrix
```
Comment (Chalarangelo) [inline snippets/transpose.md]: Please follow the instructions in CONTRIBUTING.md.

File: snippets/transpose.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
```
Comment (Chalarangelo) [inline snippets/transpose.md]: Please add a description and explanation.

File: snippets/transpose.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+```python
```
Comment (Chalarangelo) [inline snippets/transpose.md]: Use this, instead:
```suggestion
```py
```

File: snippets/transpose.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+def transpose(l):
+    return list(zip(*l))
+
```
Comment (Chalarangelo) [inline snippets/transpose.md]: You probably want to split this into two separate blocks.

File: snippets/transpose.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+l = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [10, 11, 12]]
+print(transpose(l))
+# Output: [(1, 4, 7, 10), (2, 5, 8, 11), (3, 6, 9, 12)]
```
Comment (Chalarangelo) [inline snippets/transpose.md]: You could make the example just one line long.

File: snippets/transpose.md (hunk 1 above)
Comment (HBaena) [reply snippets/transpose.md]: I'm so sorry, I thought I read it but I missed it.

I would appreciate you checking my changes.

**Pull request merged**